about/list_of_features.rst

```diff
@@ -0,0 +1,510 @@
+   - Use the same code to support both keyboards and gamepads.
+- Keyboard input.
+   - Keys can be mapped in "physical" mode to be independent of the keyboard layout.
+- Mouse input.
+   - The mouse cursor can be visible, hidden, captured or confined within the window.
+   - When captured, raw input will be used on Windows and Linux to
+     sidestep the OS' mouse acceleration settings.
+- Gamepad input (up to 8 simulatenous controllers).
+- Pen/tablet input with pressure support.
+
+Navigation
+^^^^^^^^^^
+
+- A* algorithm in 2D and 3D.
+- Navigation meshes with dynamic obstacle avoidance.
+- Generate navigation meshes from the editor.
+
+Networking
+^^^^^^^^^^
+
+- Low-level TCP networking using StreamPeer and TCP_Server.
+- Low-level UDP networking using PacketPeer and UDPServer.
+- Low-level HTTP requests using HTTPClient.
+- High-level HTTP requests using HTTPRequest.
+   - Supports HTTPS out of the box using bundled certificates.
+- High-level multiplayer API using UDP and ENet.
+   - Automatic replication using remote procedure calls (RPCs).
+   - Supports unreliable, reliable and ordered transfers.
+- WebSocket client and server, available on all platforms.
+- WebRTC client and server, available on all platforms.
+- Support for UPnP to sidestep the requirement to forward ports when hosting
+  a server behind a NAT.
+
+Internationalization
+^^^^^^^^^^^^^^^^^^^^
+
+- Full support for Unicode including emoji.
+- Store localization strings using :ref:`CSV <doc_internationalizing_games>`
+  or :ref:`gettext <doc_localization_using_gettext>`.
+- Use localized strings in your project automatically in GUI elements or by
+  using the ``tr()`` function.
+- Support for right-to-left typesetting and text shaping planned in Godot 4.0.
+
+Windowing and OS integration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Move, resize, minimize, and maximize windows spawned by the project.
+- Change the window title and icon.
+- Request attention (will cause the title bar to blink on most platforms).
+- Fullscreen mode.
+   - Doesn't use exclusive fullscreen, so the screen resolution can't be changed this way.
+     Use a Viewport with a different resolution instead.
+- Borderless windows (fullscreen or non-fullscreen).
+- Ability to keep a window always on top.
+- Transparent windows with per-pixel transparency.
+- Global menu integration on macOS.
+- Execute commands in a blocking or non-blocking manner.
+- Open file paths and URLs using default or custom protocol handlers (if registered on the system).
+- Parse custom command line arguments.
+
+Mobile
+^^^^^^
+
+- In-app purchases on Android and iOS.
+- Support for advertisements using third-party modules.
+
+Mixed reality
+^^^^^^^^^^^^^
+
+- Support for ARKit on iOS out of the box.
+- Popular VR headsets like the Oculus Quest and HTC Vive are supported thanks to plugins.
+
+GUI system
+^^^^^^^^^^
+
+Godot's small distribution size can make it a suitable alternative to frameworks
+like Electron or Qt.
```

Comment:
I moved the part about Electron/Qt at the bottom. I'm not sure about adding an introductory paragraph though, since container nodes are already described below.